app/models/user.rb

```diff
@@ -3,6 +3,9 @@ class User < ApplicationRecord
   has_many :comments, foreign_key: 'author_id', dependent: :destroy
   has_many :likes, foreign_key: 'author_id', dependent: :destroy
 
+  validates :name, presence: true
+  validates :PostsCounter, numericality: { greater_than_or_equal_to: 0 }
+
   def recent_posts
     posts.order(created_at: :desc).limit(3)
   end
```

Comment:
- [ ]  Good job with test coverage 💪, but since we are aiming for 100% test coverage, and it is also a [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md) for this project, "Add unit specs for all of your models' methods and validations", please add a unit test for `recent_posts` method.